.bazelversion

```diff
@@ -0,0 +1 @@
+7.0.0
```

Comment:
Bump to 7.1.2, if it is the version everywhere.

Reply:
Nope. 7.0.0 inside docker, gotta keep it like that for now.